### DISCUSSION TITLE
Want to add an APP from Google Play Store

### DISCUSSION THREAD

**Discussion by KuSantosh100** [ImranR98/Obtainium#1731] [Q&A] [answered]
How can I add an APP from Google Play store?
We have the app share option from Google PS, is there any possibility to add it into the list?

It's for basic use purpose like adding to wishlist, like bookmark 🔖 that app

**Comment by DwainZwerg** [comment]
Currently, the Google Play Store cannot be used as an app source. See https://github.com/ImranR98/Obtainium/issues/343

**Comment by KuSantosh100** [comment]
@DwainZwerg  hey thanks for the response. I have looked in it. 
What I really wanted is that a feature where I can bookmark apps which I have uninstalled and liked apps into different categories for referrence and use in a wihslist, Can you suggest me for that?

**Reply by DwainZwerg** [reply, marked as answer]
@KuSantosh100 You could submit to the issue [TrackOnlyGooglePlay](https://github.com/ImranR98/Obtainium/issues/212). The feature request is thematically similar and yours would be fulfilled by completing this issue.
